plugins/uefi-dbx/fu-self-test.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupd.h>
+
+#include "fu-uefi-dbx-common.h"
+#include "fu-uefi-dbx-file.h"
+
+static void
+fu_uefi_dbx_file_parse_func (void)
+{
+	gboolean ret;
+	gsize bufsz = 0;
+	g_autofree gchar *fn = NULL;
+	g_autofree guint8 *buf = NULL;
+	g_autoptr(FuUefiDbxFile) uefi_dbx_file = NULL;
+	g_autoptr(GError) error = NULL;
+
+	/* load file */
+	fn = fu_uefi_dbx_get_dbxupdate (&error);
```

Comment:
Rather than splitting out the function to it's own file, why not just load the plugin and run it's startup routine?

Reply:
I was pondering fuzzing it.

Reply:
Can't you still fuzz if you have it set to obey paths set via environment variable and loading plug-in?

Reply:
Loading the plugin dynamically is an order of magnitude slower.

Reply:
I'm just adding the fuzzer bits now.

Reply:
I guess that's true, a lot more code is loaded.  OK well no concerns then, fuzzing is a good thing.